src/pages/RoomPage/RoomViews/TestInfo/TestInfo.tsx

```diff
@@ -0,0 +1,148 @@
+import { AttachmentRounded } from '@mui/icons-material';
+import { useMutation } from '@tanstack/react-query';
+import { useMemo } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { Button, Icon } from '@/components';
+import { useCustomTheme } from '@/hooks/useCustomTheme';
+import * as S from '@/pages/RoomPage/RoomPage.style';
+import { PATH } from '@/routes/path';
+import { startTest } from '@/services/Room/Room';
+import useMeStore from '@/store/Me';
+import useRoomStore from '@/store/Room';
+import { MemberType } from '@/types/room';
+
+interface TestInfoProps {
+  className: string;
+  myRoomData: MemberType;
+}
+
+export default function TestInfo({ className, myRoomData }: TestInfoProps) {
+  const { theme } = useCustomTheme();
+
+  const { roomData, setRoomData } = useRoomStore();
+  const { timeLimit, problemLink, roomId, members } = roomData;
+  const { me } = useMeStore();
+
+  const navigate = useNavigate();
+
+  const isTestReady = useMemo(() => {
+    const result = members.findIndex(member => member.ready === false);
+    return result === -1 ? true : false;
+  }, [members]);
+
+  const calcTime = useMemo(() => {
+    const hours = Math.floor(timeLimit / 60);
+    const minutes = Math.floor(timeLimit % 60);
+
+    return { hours, minutes };
+  }, [timeLimit]);
+
+  const changeMemberData = (newData: Partial<MemberType>) => {
+    const myIndex = members.findIndex(member => member.id === me.id);
+
+    const updatedData = [...members];
+    updatedData[myIndex] = {
+      ...updatedData[myIndex],
+      ...newData,
+    };
+
+    setRoomData({ ...roomData, members: updatedData });
+  };
+
+  const mutation = useMutation({
+    mutationFn: startTest,
+    onError: () => {
+      alert('테스트를 시작하지 못했습니다.');
+
+      navigate(`${PATH.ROOM}/${roomId}`);
+    },
+  });
+
+  const handleStartTest = async () => {
+    await mutation.mutate(`/${roomId}`);
```

Comment:
p5)
추가적으로 방장 혼자 있을 때 테스트 시작을 누르면 혼자 풀기를 할 것인지 묻는 확인 로직도 있으면 좋을 거 같네요!
```suggestion
    await mutation.mutate(`/${roomId}`);
    if (members.length === 1) {
      // 확인 로직 추가, 모달 or alert
      alert('정말 혼자 풀꺼야?');
    }
```

Reply:
```
const isTestReady = useMemo(() => {
  const result = members.findIndex(member => member.ready === false);
  return result === -1 ? true : false;
}, [members]);
```
해당 로직은 29번줄 isTestReady에서 체크를 하고 있고 "테스트 시작"이 활성화되어 있는 상태라서 인지가 가능할 것 같아 추가하지 않았습니다!

테스트 시작 전 alert가 필요하다면 handleStartTest에서 "바로 테스트를 시작하시겠습니까?"라는 문구로 대체하면 좋을 것 같아요! 5초 뒤 바로 테스트 시작도 괜찮은 것 같습니다.